timelock-server/src/main/java/com/palantir/atlasdb/timelock/TimeLockServerLauncher.java

```diff
@@ -52,7 +52,6 @@ public void initialize(Bootstrap<TimeLockServerConfiguration> bootstrap) {
     public void run(TimeLockServerConfiguration configuration, Environment environment) {
         environment.getObjectMapper().registerModule(new Jdk8Module());
         environment.jersey().register(HttpRemotingJerseyFeature.INSTANCE);
-        environment.jersey().register(new QosServiceResource());
```

Comment:
this will now be covered by internal tooling stuff, I guess?

Reply:
Yes and also the `TimelockServerLauncher` isnt aware of the `QosServiceRuntimeConfig`. We can have a separate launcher or make TLSL aware of the config for local testing I think.